src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -59,10 +60,10 @@ class JobWrapper:
     """The only user of the JobWrapper is the JobWrapperTemplate"""
 
     #############################################################################
-    def __init__(self, jobID=None, jobReport=None):
+    def __init__(self, jobID: int | None = None, jobReport: JobReport | None = None):
         """Standard constructor"""
         self.initialTiming = os.times()
-        self.section = os.path.join(getSystemSection("WorkloadManagement/JobWrapper"), "JobWrapper")
+        self.section = "/Systems/WorkloadManagement/JobWrapper"
```

Comment:
This is risky. I see why you do it but we can't rely the CS structure will be already updated by the time this goes in production.

Reply:
I reverted the change